packages/cfpb-layout/src/molecules/heroes.less

```diff
@@ -164,4 +164,69 @@
             }
         } );
     }
+
+    &__jumbo {
+        background-color: @white;
+
+        .m-hero_subhead {
+            .lead-paragraph();
+        }
+
+        .respond-to-max( @bp-xs-max, {
+            .m-hero_wrapper {
+                // Remove wrapper bottom padding.
+                padding-bottom: 0;
+            }
+
+            .m-hero_image-wrapper {
+                position: relative;
+                border-width: 0;
+            }
+        } );
+
+        .respond-to-min( @bp-sm-min, {
+            .m-hero_wrapper {
+                // Position the hero image all the way to the right plus a little
+                // extra so that it flows into the gutter
+                background-position: calc( 100% + @grid_gutter-width ) center;
+                background-repeat: no-repeat;
+                background-size: 50%;
+                border-right: 1px solid @gray-40;
+                border-bottom: 1px solid @gray-40;
+                border-left: 1px solid @gray-40;
+            }
+
+            .m-hero_text {
+                .grid_column( 6, 12 );
+            }
+
+            .m-hero_image-wrapper {
+                .grid_column( 6, 12 );
+            }
+
+            .m-hero_image {
+                display: none;
+            }
+
+            .m-hero_heading {
+                .heading-1();
```

Comment:
Hmmm why do we have `.heading-1()` here and `h1` in the markup?

Reply:
Hmmm it might have been added for clarity but yeah it's redundant. I'll remove it.

Reply:
We might actually keep it here. Hear me out. So in the markup we have `h1 m-hero_heading` and `m-hero_subhead`. The `m-hero_subhead` class only sets the `.lead-paragraph()` mixin. So why isn't `lead-paragraph` just in the markup instead of `m-hero_subhead`? We could do it that way, although organizationally it's clearer to have these named BEM classes in the markup for breaking up the component parts. That means we should have `m-hero_heading` and `m-hero_subhead` only though, and `h1` should be set in the CSS via `.heading-1()`. Within the same component, it seems like we should have all the class rules in the CSS, or all in the markup. What do you think?